test/browser_tests/page_objects/page_about-us.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function AboutUs() {
+  this.get = function() {
+    browser.get( '/about-us/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+  this.activityBlock = element.all( by.css( 'aside.activity' ) );
```

Comment:
I think you're right. I'll run it without the aside to confirm, but that's a good catch.